commons/src/main/kotlin/com/simplemobiletools/commons/helpers/ContactsConfig.kt

```diff
@@ -0,0 +1,77 @@
+package com.simplemobiletools.commons.helpers
+
+import android.content.Context
+
+class ContactsConfig(context: Context) : BaseConfig(context) {
```

Comment:
use the causal config file for these, remove ContactsConfig

Reply:
Hey man, which config file are you referring to here?